features/foutafhandeling.feature

```diff
@@ -0,0 +1,292 @@
+        | detail | Precies 1 parameter van adresseerbaarObjectIdentificatie, nummeraanduidingIdentificatie, locatie of bbox moet worden opgegeven |
+        En bevat de response geen invalidParams
+
+Rule: fields parameter bevat geen onbekende kenmerk namen
+
+    Scenario: Er is één onbekende kenmerk naam opgegeven
+        Als '/adressen?pandIdentificatie=0345100002016017&fields=bestaatniet' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name   | code   | reason                                                    |
+        | fields | fields | Deel van de parameterwaarde is niet correct: bestaatniet. |
+
+    Scenario: Er zijn meerdere onbekende kenmerk namen opgegeven
+        Als '/adressen?pandIdentificatie=0345100002016017&fields=bestaatniet,bestaatookniet' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name   | code   | reason                                                       |
+        | fields | fields | Deel van de parameterwaarde is niet correct: bestaatniet.    |
+        | fields | fields | Deel van de parameterwaarde is niet correct: bestaatookniet. |
+
+Rule: expand parameter bevat geen onbekende resource namen
+
+    Scenario: Er is één onbekende resource naam opgegeven
+        Als '/adressen?zoekresultaatIdentificatie=adr-f6b00930705c1ef12c2624623a797d21&expand=bestaatniet' wordt aangeroepen
+        Dan bevat de response de volgende kenmerken
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name   | code   | reason                                                    |
+        | expand | expand | Deel van de parameterwaarde is niet correct: bestaatniet. |
+
+    Scenario: Er zijn meerdere onbekende resource namen opgegeven
+        Als '/adressen?zoekresultaatIdentificatie=adr-f6b00930705c1ef12c2624623a797d21&expand=bestaatniet' wordt aangeroepen
```

Comment:
dan moet dit zijn: expand=bestaatniet,bestaatookniet

wellicht nuttig om te illustreren hoe niet alleen onbekende resource, maar ook onbekend veld in een resource kan worden gemeld. Bijvoorbeeld expand=bestaatniet,nummeraanduiding.bestaatookniet

Reply:
aangepast. En neem ik op als extra scenario